internal/sops/keyservice/server.go

```diff
@@ -86,6 +106,138 @@ func (ks *Server) decryptWithAge(key *keyservice.AgeKey, ciphertext []byte) ([]b
 	return plaintext, err
 }
 
+func (ks *Server) newKeyvaultAuthorizerFromFile(fileLocation string) (autorest.Authorizer, error) {
```

Comment:
Instead of embedding this logic into the Server, this should move to a separate `azure` package within `sops`, as done for the other key services.

Reply:
I have named the package azure_kv for consistancy, but ok.

Reply:
Can you name it `azkv` in that case, that aligns with SOPS itself: https://github.com/mozilla/sops/tree/master/azkv